docs/release_notes/v1.0.0-rc.1.md

```diff
@@ -0,0 +1,397 @@
+  
+# Dapr 1.0.0-rc.1
+
+We're happy to announce the release of Dapr 1.0.0-rc.1!
+
+We would like to extend our thanks to all the new and existing contributors who helped make this release happen.
+
+**Highlights**
+
+This is a release candidate version as we progress towards Dapr v1.0.0. We appreciate you trying Dapr and providing feedback on this release.
+
+> Dapr CLI version 1.0.0-rc.1 is not compatible with previous runtime versions. Download Dapr CLI 1.0.0-rc.1 from [here](https://github.com/dapr/cli/releases) and put the `dapr` binary in your PATH.
+
+> For Java applications, use SDK version 0.10.0-rc-1.
```

Comment:
I would rephrase so that it doesn't seem like you must have an SDK.

Something like `For the Java SDK, use version 0.10.0-rc-1`.

Same thing for .Net below.

Reply:
It is very unlikely for users not to use SDK to use Dapr in these 2 languages. I will rephrase it.